tests/integration/aof.tcl

```diff
@@ -491,9 +491,11 @@ tags {"aof external:skip"} {
             set start [clock clicks -milliseconds]
```

Comment:
looking at this test now, i wonder if and how come it achieves what it aims for.
doesn't the enabling of the `appendonly` config trigger an AOFRW?
i.e. in which case it'll override the AOF file and eliminate the script in it?
maybe instead we just wanna start redis from that AOF file (`appendonly` enabled on startup)?

Reply:
You are right. The previous test is indeed wrong. After the rewrite, those lua script are gone, I will fix it by the way.